src/App.css

```diff
@@ -13,6 +13,65 @@ html, body, #root, #app {
   font-size: 16px;
 }
 
+.toolbar-container {
+  text-align: center;
+  position: fixed;
+  top: 10px;
+  right: 10px;
+  z-index: 2;
+  -webkit-touch-callout:none;
+  -webkit-user-select:none;
+  -khtml-user-select:none;
+  -moz-user-select:none;
+  -ms-user-select:none;
+  user-select:none;
+  -webkit-tap-highlight-color:rgba(0,0,0,0);
+}
+
+.toolbar {
+  touch-action: inherit;
+  display: block;
+  text-align: right;
+}
+
+.toolbar-overlay {
+  touch-action: inherit;
+  display: inline-block;
+  width: 120px;
```

Comment:
Fixed width won't work well here since the content may vary in length.

Reply:
True, but I was saving it for the scroll milestone.

Reply:
Okay